File: Agosto - Introduccion/Introducción POO/py/persona.py
class Persona:
    def __init__(self, nombre, edad, carrera):
        self.nombre = nombre
        self.edad = edad
        self.carrera = carrera
    
    def saludar(self):
        print (f"hola mi nombre es {self.nombre} y tengo {self.edad} años.")
        
    def cumpleaños(self):
        self.edad += 1
        print (f" ¡Feliz cumpleaños! ahora tengo {self.edad} años.")
    
    def estudiar (self):
        print (f"Estoy estudiando {self.carrera}.")

#crear una instancia de la clase persona
persona1 = Persona ("Juan", 20, "Medicina")

#llamar a los metodos
persona1.saludar()
persona1.estudiar()

File: Agosto - Introduccion/Agosto - Introduccion/Introducción POO/js/persona.js
class Persona {
    constructor(nombre, edad, carrera) {
        this.nombre = nombre;
        this.edad = edad;
        this.carrera = carrera;
    }

    saludar() {
        console.log('hola, mi nombre es' + this.nombre + 'y tengo' + this.edad + 'años');
    }

    cumpleaños() {
        this.edad = this.edad + 1;
        console.log('¡Feliz cumpleaños! Ahora tengo' + this.edad + 'años');
    }

    estudiar() {
        console.log('Estoy estudiando' + this.carrera);
    }
}

//crear una instancia
const persona1 = new Persona("Ana", 22, "Ingenieria Informatica" );

//Llamar a los métodos
persona1.saludar();
persona1.estudiar();